lib/google-music-electron.js

```diff
@@ -216,6 +226,13 @@ function launchGme() {
     gme.browserWindow.on('minimize', gme.toggleVisibility);
   }
 
+  // When we recieve the close event, before we clean up, save the current size and position of the window
+  // DEV: Note this fires when we still have access to the browserWindow, before 'closed'
+  gme.browserWindow.on('close', function handleCloseEvent () {
+    logger.debug('Setting window size in config.');
+    config.set('window-info', gme.browserWindow.getBounds());
+  });
```

Comment:
Good point! I did attempt this first using the resize and move events, but as you mention the performance issues were a little hairy, it also of course spams up the log if you leave that log statement in there. I'll admit I was unaware of `_.debounce` but it definitely seems like a good approach.

The `resize` and `move` events seem like much more natural places to trigger this change, too, so I'll give this a shot shortly.